CSharp_Course_Project/Student.cs

```diff
@@ -0,0 +1,131 @@
+﻿using CSharp_Course_Project.Task4_Stack;
+using System;
+using System.Collections.Generic;
+
+namespace CSharp_Course_Project.Task1_StudentList
+{
+    public class Student
+    {
+        private string name;
+
+        private string surname;
+
+        private byte course;
+
+        private string group;
+
+        private string faculty;
+
+        public Queue<string> TaskList { get; set; }
+
+        public Stack<string> bookList = new Stack<string>();
```

Comment:
Please move to constructor.

Reply:
Public field?????????
????????????????????

Reply:
fixed)))